taxi/views.py

```diff
@@ -44,6 +66,27 @@
     model = Car
 
 
+class CarCreateView(LoginRequiredMixin, generic.edit.CreateView):
+    model = Car
+    fields = "__all__"
+    success_url = reverse_lazy("car-list")
+    template_name = "taxi/car_form.html"
+
+
+class CarUpdateView(LoginRequiredMixin, generic.edit.UpdateView):
+    model = Car
+    fields = "__all__"
+    success_url = reverse_lazy("car-list")
+    template_name = "taxi/car_form.html"
+
+
+class CarDeleteView(LoginRequiredMixin, generic.edit.DeleteView):
+    model = Car
+    fields = "__all__"
```

Comment:
The `fields` attribute is not applicable for `DeleteView`. You should remove this line from `CarDeleteView`.